mentoring/mentor-guide.md

```diff
@@ -94,6 +94,12 @@ A former GSoC intern for Kubernetes is now leading this for us - mentoring works
 * Typically a one hour a quarter time commitment is expected; this is an opt-in sign up.
 * You will need to be on the call 5 minutes early to test audio quality and make sure we are good to go for streaming.
 
+*[Pod Mentoring](mentoring-events.md)*
+* Short group sessions with other mentors during KubeCon/CloudNativeCon events answering mentee questions
+  * Setup to eleminate awkwardness of 1:1s and ensure a wider breadth of knowledge to reduce pressure of having to provide an answer
```

Comment:
I think in this case set up might be two words, but I actually think maybe it might sound better as "Designed to eliminate the awkwardness of ..." Also, there is a typo in the spelling of eliminate :)

Reply:
Fixed!